request_a_govuk_domain/request/models/review.py

```diff
@@ -18,91 +31,90 @@ class Review(models.Model):
     registrant_person_notes = models.TextField(
-        max_length=NOTES_MAX_LENGTH, blank=True, null=True
+        max_length=NOTES_MAX_LENGTH, validators=[MinLengthValidator(NOTES_MIN_LENGTH)]
     )
 
     registrant_permission = models.CharField(
-        choices=review_choices.RegistrantPermissionReviewChoices.choices,
-        blank=True,
-        null=True,
+        choices=RegistrantPermissionReviewChoices.choices
     )
     registrant_permission_notes = models.TextField(
-        max_length=NOTES_MAX_LENGTH, blank=True, null=True
+        max_length=NOTES_MAX_LENGTH, validators=[MinLengthValidator(NOTES_MIN_LENGTH)]
     )
 
     policy_exemption = models.CharField(
-        choices=review_choices.PolicyExemptionReviewChoices.choices,
+        choices=PolicyExemptionReviewChoices.choices,
         blank=True,
         null=True,
     )
     policy_exemption_notes = models.TextField(
         max_length=NOTES_MAX_LENGTH, blank=True, null=True
     )
 
-    domain_name_rules = models.CharField(
-        choices=review_choices.DomainNameRulesReviewChoices.choices,
-        blank=True,
-        null=True,
-    )
+    domain_name_rules = models.CharField(choices=DomainNameRulesReviewChoices.choices)
     domain_name_rules_notes = models.TextField(
-        max_length=NOTES_MAX_LENGTH, blank=True, null=True
+        max_length=NOTES_MAX_LENGTH, validators=[MinLengthValidator(NOTES_MIN_LENGTH)]
     )
 
     registrant_senior_support = models.CharField(
-        choices=review_choices.RegistrantSeniorSupportReviewChoices.choices,
+        choices=RegistrantSeniorSupportReviewChoices.choices,
         blank=True,
         null=True,
     )
     registrant_senior_support_notes = models.TextField(
         max_length=NOTES_MAX_LENGTH, blank=True, null=True
     )
 
-    registry_details = models.CharField(
-        choices=review_choices.RegistryDetailsReviewChoices.choices,
-        blank=True,
-        null=True,
-    )
+    registry_details = models.CharField(choices=RegistryDetailsReviewChoices.choices)
     registry_details_notes = models.TextField(
-        max_length=NOTES_MAX_LENGTH, blank=True, null=True
+        max_length=NOTES_MAX_LENGTH, validators=[MinLengthValidator(NOTES_MIN_LENGTH)]
     )
 
-    reason = models.TextField(max_length=NOTES_MAX_LENGTH, blank=True, null=True)
+    reason = models.TextField(
+        max_length=NOTES_MAX_LENGTH, validators=[MinLengthValidator(NOTES_MIN_LENGTH)]
+    )
 
     history = HistoricalRecords()
 
     def is_approvable(self) -> bool:
-        return True
+        if (
+            self.registrar_details == RegistrarDetailsReviewChoices.APPROVE
+            and self.domain_name_availability
+            == DomainNameAvailabilityReviewChoices.APPROVE
+            and self.registrant_org == RegistrantOrgReviewChoices.APPROVE
+            and self.registrant_person == RegistrantPersonReviewChoices.APPROVE
+            and (
```

Comment:
Do we need one more condition for the exemption file as we upload two files in some paths?

Reply:
Done for the other paths now.